drivers/include/isl29020.h

```diff
@@ -31,12 +31,22 @@
 extern "C" {
 #endif
 
- /**
-  * @brief   The sensors default I2C address
-  */
+/**
+ * @defgroup drivers_isl29020_config     ISL29020 light sensor driver compile configuration
+ * @ingroup config_drivers_sensors
+ * @{
+ */
+/**
+ * @brief Default address
```

Comment:
Also could indicate that this default address corresponds to `A0` connected to GND.

Reply:
Done. Thanks